Framework/DataHandling/src/SaveStl.cpp

```diff
@@ -0,0 +1,118 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "MantidDataHandling/SaveStl.h"
+#include "MantidGeometry/Objects/MeshObject.h"
+#include "MantidKernel/DateAndTime.h"
+
+#include <Poco/File.h>
+#include <fstream>
+#include <vector>
+namespace Mantid {
+namespace DataHandling {
+enum class ScaleUnits { metres, centimetres, millimetres };
+
+namespace {
+/**
```

Comment:
Should this comment be attached to the class in the header. It looks like it is not describing the method below?